packages/ui/src/styles/_root.sass

```diff
@@ -27,15 +27,18 @@ $cui-buildState-alreadyOutputRoot: false !default
 		margin: 0
 		padding: 0
 
+	html
+		display: flex
+		flex-direction: column
+		min-width: fit-content
```

Comment:
Proč je tohle potřeba? Co by se stalo, kdyby se to nenastavilo?

Reply:
I've alredy updated the code. Only the `display:flex` is necessary, since the default flex-direction (row) handles the rest.

`<html>` would not expand horizontally, `<body>` would not extend either causing sticky container with menu loose stickiness when content is scrolled further to side pass the body boundary:

without `display:flex` (bar scrolled away) | with `display:flex` (bar with working `position:sticky`)
----------------------|------------------
<img width="772" alt="Screenshot 2022-02-07 at 14 40 22" src="https://user-images.githubusercontent.com/387611/152799624-77062ce1-0111-4aae-9ee9-ee465621f326.png"> | <img width="771" alt="Screenshot 2022-02-07 at 14 40 33" src="https://user-images.githubusercontent.com/387611/152799623-e02b4b85-2f14-47b7-bebe-56485b423b79.png">